.meta.toml

```diff
@@ -31,6 +31,20 @@ additional-config = [
 [tox]
 testenv-deps = [
     "cffi",
+    "# The universal2 environment is for Python on macOS built with",
+    "# universal2 mode instead of architecture-specific. These dependencies",
+    "# must be installed separately, zc.buildout is incompatible with the",
+    "# universal2 mode and cannot install them itself.",
+    "universal2: zope.interface",
+    "universal2: zope.security",
+    "universal2: zope.container",
+    "universal2: Persistence",
+    "universal2: Acquisition",
+    "universal2: AccessControl",
+    "universal2: zodbpickle",
+    "universal2: charset_normalizer",
+    "universal2: MarkupSafe",
+    "universal2: zope.testrunner",
```

Comment:
`zope.testrunner` does not contain any C code, it should not have a `universal2` wheel.

Reply:
This is not about creating a universal2 wheel for publishing. It's about running in a universal2 environment, which the macOS GHA runners are.